app/views/categories/_category.html.erb

```diff
@@ -5,7 +5,7 @@
         <div class="grid content-around grid-cols-2 grid-rows-2 gap-4">
           <% category.podcasts.sample(4).each do |podcast| %>
             <div>
-              <% if podcast %>
+              <% if podcast && podcast.image_urls.any? %>
```

Comment:
good touch thanks. I'd like to say, here we have a diffent problem. When a user signs up, we fetch podcasts, add new podcasts, and their images to db. If fetching process hasn't finished in background job yet, we cannot show the podcast image here.

Reply:
Let's keep it.